pkg/minikube/cluster/cluster_windows_test.go

```diff
@@ -36,16 +36,9 @@ func TestCreateHypervHostGeneratesCorrectIsoUrl(t *testing.T) {
 	isoPath := filepath.Join(constants.Minipath, "cache", "iso", "boot2docker.iso")
 	expectedURL := "file://" + filepath.ToSlash(isoPath)
 
-	d := createHypervHost(machineConfig)
-	expectedDriver := "*hyperv.Driver"
+	options, _ := createHypervHost(machineConfig)
```

Comment:
@hferentschik you need to remove `github.com/docker/machine/drivers/hyperv` and `reflect` package as they are not required now and reason for test failure in appveyor.